src/store/messagesStore.js

```diff
@@ -515,6 +540,13 @@ const actions = {
 			})
 		}
 
+		if (message.systemMessage === 'history_cleared') {
```

Comment:
but what if you just scrolled back in the history until this message, it would drop all chat messages?

Reply:
basically we should only clear older messages

Reply:
Ah, already the case